packages/protocol/contracts/ZKERC20/ZKERC20.sol

```diff
@@ -200,14 +202,14 @@ contract ZKERC20 {
     }
 
     function logInputNotes(bytes memory inputNotes) internal {
-        for (uint i = 0; i < inputNotes.getLength(); i += 1) {
+        for (uint i = 0; i < inputNotes.getLength(); i = i.add(1)) {
```

Comment:
TODO: we should discuss whether using safeMath for iterations like these is redundant